yarn.lock

```diff
@@ -925,6 +925,11 @@
   resolved "https://registry.yarnpkg.com/@endo/base64/-/base64-0.2.27.tgz#6c68fe83092c621e2d46f77523e0e898e7f7fa68"
   integrity sha512-5LVFV7EDZemO+G8plo3pENFNmuCbo4G3Mnj8meRh9P7VyYlRK3QSvUe7QZ59BUb88iByPd6h0/a2PWyUC+zucw==
 
+"@endo/base64@^0.2.29":
```

Comment:
there are two versions in the lockfile now. we should have just one. `npx yarn-deduplicate` might do the job. If not we may need to bump all `@endo` deps to latest

Reply:
I think that did the trick, PTAL